src/framework/preferences/js/LocalizationEnactor.js

```diff
@@ -35,7 +35,10 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
             localeChange: {
                 checks: {
                     inPanel: {
```

Comment:
What's the "inPanel" check for?

Reply:
It's to see if we are running in the separated panel's iframe. I'm going to add a comment about that.